app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  has_many :attendees
+  has_many :events, through: :attendees
+  has_many :user_events, :foreign_key => "user_id", :class_name => "Event" 
```

Comment:
You may need to refactor line 8 to reflect this below:
has_many :events, foreign_key: :creator_id